### DISCUSSION TITLE
运行的官方demo效果这么差么，这识别的都是啥呀，还是我的用法不对

### DISCUSSION THREAD

**Discussion by WusamX** [PaddlePaddle/PaddleOCR#13049] [Q&A]
<img width="793" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/91404cad-083c-4a49-af32-e057c67e8104">
<img width="692" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/88de79f3-05b8-4dce-8f6e-1b1d2c0d3764">
<img width="662" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/fd1aca42-160f-486c-8365-8349272792fd">
<img width="935" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/5ccd9942-e75b-49b8-adf3-337296dbabc6">
<img width="356" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/9c463d5f-80b7-4738-a64d-3529dc2ec688">
<img width="1129" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20549190/073a8529-c0f4-42bd-8382-60e3094e3c5c">

**Comment by SWHL** [comment]
试一下paddlepaddle cpu版，看看

**Comment by GreatV** [comment]
You may need to change `image_orientation=False`. This may be a bug, we will fix it.

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/4bc47d1e-1302-494b-85d3-87f7fa665c67)

**Comment by jingsongliujing** [comment]
其实是你用了方向分类，然后巧合的是刚好图像分类错了，就可视化的时候，你先旋转180度，再画这些可视化的框，就对上了
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/45508593/ca1b43af-54ab-4a6b-99a0-024b8cb322c3)
```python
import os
import cv2
from paddleocr import PPStructure, draw_structure_result, save_structure_res
from PIL import Image

def Structure_analysis(img_path):
    table_engine = PPStructure(show_log=True, image_orientation=True)
    save_folder = './output/table'
    img = cv2.imread(img_path)
    result = table_engine(img)
    save_structure_res(result, save_folder, os.path.basename(img_path).split('.')[0])

    for line in result:
        line.pop('img')
        print(line)

    font_path = './doc/fonts/simfang.ttf'  # PaddleOCR下提供字体包
    image = Image.open(img_path).convert('RGB')
    image=image.rotate(180)  # 旋转图片
    im_show = draw_structure_result(image, result, font_path=font_path)
    im_show = Image.fromarray(im_show)
    im_show.save('result.jpg')
    pass

if __name__ == '__main__':
    Structure_analysis('./ppstructure/docs/table/1.png')
    pass
